scripts/nmt/nmt.py

```diff
@@ -101,11 +139,11 @@ def request(inputs,args):
                     request(batch, args)
                     batch = []
             if len(batch) > 0:
-                request(batch, args)
+                request(batch, args, dnt_phrases_input)
```

Comment:
can we read it from args itself inside request() function? instead of passing it additionally.
line 130 can be done inside request() function